guides/source/upgrading_ruby_on_rails.md

```diff
@@ -70,6 +70,66 @@ Upgrading from Rails 5.0 to Rails 5.1
 
 For more information on changes made to Rails 5.1 please see the [release notes](5_1_release_notes.html).
 
+### Moving from Erubis to Erubi
```

Comment:
I don't think this needs a mention here; a note in Incompatibilities in the release notes should be sufficient.

Approximately zero applications are going to be affected by this, but I worry giving it too-prominent placement will lead people to assume it applies to them just because they're using `.erb` templates.